src/apps/favorites-list/FavoritesList.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useEffect } from "react";
+import EmptyList from "../../components/empty-list/empty-list";
+import usePager from "../../components/result-pager/use-pager";
+import { useGetList } from "../../core/material-list-api/material-list";
+import { useText } from "../../core/utils/text";
+import { Pid } from "../../core/utils/types/ids";
+import SearchResultListItemAdapter from "./materials/SearchResultListItemAdapter";
+
+export interface FavoritesListProps {
+  pageSize: number;
+}
+
+const FavoritesList: React.FC<FavoritesListProps> = ({ pageSize }) => {
+  const t = useText();
+  const { data } = useGetList("default");
+  const [displayedMaterials, setDisplayedMaterials] = useState<Pid[]>([]);
+  const [materials, setMaterials] = useState<Pid[]>([]);
+  const { itemsShown, PagerComponent } = usePager(materials.length, pageSize);
+
+  useEffect(
+    () => setDisplayedMaterials([...materials].splice(0, itemsShown)),
+    [itemsShown, materials]
+  );
+
+  useEffect(() => {
+    if (data && data.collections) {
+      setMaterials(data.collections as Pid[]);
+    }
+  }, [data]);
+
+  return (
+    <div className="search-result-page">
+      <h1 className="text-header-h2 mb-16 search-result-title">
+        {t("favoritesListHeaderText")}
+      </h1>
+      {materials.length > 0 && (
+        <p className="text-small-caption my-32">
+          {t("favoritesListMaterialsText", {
+            placeholders: { "@count": materials.length }
+          })}
+        </p>
+      )}
+      {displayedMaterials.length > 0 && (
+        <ul className="search-result-page__list my-32">
+          {displayedMaterials.map((pid) => (
+            <li key={pid}>
+              <SearchResultListItemAdapter key={pid} pid={pid} />
```

Comment:
I am sorry but I don't think you can go this way.
I agree in that the search result list and the favourites list have so much similarties that they both can be covered by the same model.
But! It has to be solved then from the ground up by generalise the list both in dpl-design-system and in dpl-react  by calling it something generic like: "card list" or similar. And then use it both for the search results and for the favourites list.
I know it takes some extra work. But that is the way to do it in order to do it to keep the code bases maintainable in the future.